pkg/apis/serving/v1alpha2/inferenceservice_types.go

```diff
@@ -197,6 +197,8 @@ type XGBoostSpec struct {
 	RuntimeVersion string `json:"runtimeVersion,omitempty"`
 	// Defaults to requests and limits of 1CPU, 2Gb MEM.
 	Resources v1.ResourceRequirements `json:"resources,omitempty"`
+	// The prediction method. Defaults to "predict".
+	Method string `json:"method,omitempty"`
```

Comment:
@ellistarn any thoughts here? maybe we should have a model server parameter section ?